Loop/Extensions/DeviceDataManager+DeviceStatus.swift

```diff
@@ -71,7 +71,7 @@ extension DeviceDataManager {
         {
             return .openAppURL(url)
         } else if let cgmManagerUI = (cgmManager as? CGMManagerUI) {
-            return .presentViewController(cgmManagerUI.settingsViewController(for: displayGlucoseUnitObservable, bluetoothProvider: bluetoothProvider, colorPalette: .default))
+            return .presentViewController(cgmManagerUI.settingsViewController(bluetoothProvider: bluetoothProvider, displayGlucoseUnitObservable: displayGlucoseUnitObservable, colorPalette: .default))
```

Comment:
Dexcom plugin will need updating as well, eh?

Reply:
Just looked at the LW PR and see the plugin updates there.

Reply:
Yep, for these wide ranging changes I submit `LoopKit` and `Loop` PRs first to make sure the base code and protocol changes are acceptable. After they are approved, I'll submit the rest of the PRs for review.